tests/workflows/basic_parent_context/tests/test_basic_wrapper.py

```diff
@@ -0,0 +1,36 @@
+from uuid import uuid4
+
+from vellum.core.context import get_execution_context, wrapper_execution_parent_context
+from vellum.workflows.events.types import NodeParentContext, ParentContext
+
+from tests.workflows.basic_parent_context.basic_workflow import StartNode
+
+
+class A:
+    def __init__(self):
+        self._span_id = uuid4()
+        self._context = None
+
+    @wrapper_execution_parent_context(
+        lambda self: NodeParentContext(node_definition=StartNode().__class__, span_id=self._span_id))
```

Comment:
[blocking] we should only initialize node's during the workflow runner. here, we could easily simplify to just `node_definition=StartNode`